examples/Searching for Local Environments in complex crystal.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
I'd rename to `Identifying Local Environments in a Complex Crystal.ipynb` and add a Markdown cell at the top with the title. See [example](https://freud.readthedocs.io/en/latest/gettingstarted/examples/examples/Using%20Machine%20Learning%20for%20Structural%20Identification.html). The Markdown heading level (`# Heading` vs `## Heading`) is important for the documentation to render correctly.

Reply:
Should look like this: https://github.com/glotzerlab/freud-examples/blame/master/examples/Handling%20Multiple%20Particle%20Types%20(A-B%20Bonds).ipynb#L1-L11